packages/core/http/core-http-integration-tests/lifecycle_handlers.test.ts

```diff
@@ -103,6 +103,9 @@ describe('core lifecycle handlers', () => {
     const testRoute = '/version_check/test/route';
 
     beforeEach(async () => {
+      env = Env.createDefault(REPO_ROOT, getEnvOptions());
```

Comment:
We can't read the main kibana `package.json` from within another package. I changed this to get the kibana version from env.